src/test/java/com/epam/dataservice/PoliceForceServiceTestMock.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.dataservice;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PoliceForceServiceTestMock {
+
+	@Mock
+	PoliceForceService policeForceService;
+
+	@Test
+	public void testGetContactNo() {
+		Mockito.when(policeForceService.getContactNo("North Yorkshire")).thenReturn("1234");
+		Mockito.when(policeForceService.getContactNo("North Yorkshire1")).thenReturn("123");

```

Comment:
You testing Mockito here, but not a service.   Mock ideal if you have method, that call something else - and you will mock other methods, but not tested one.